### DISCUSSION TITLE
Mullvad-CLI Package for Edgerouter

### DISCUSSION THREAD

**Discussion by ghost** [mullvad/mullvadvpn-app#9096] [Feature requests & Ideas]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

Essentially the same as #6008.
I would like a Mullvad CLI package for EdgeRouter-X.

### Alternative solutions

none

### Type of feature

- [ ] Better privacy/anonymity
- [ ] Better at circumventing censorship
- [X] Easier to use
- [ ] Other

### Operating System

- [ ] Android
- [ ] iOS
- [ ] Windows
- [ ] macOS
- [X] Linux

**Comment by MarkusPettersson98** [comment]
Hello, thanks for reaching out to us with this feature suggestion! I will lift this internally :blush:

**Comment by MarkusPettersson98** [comment]
Hi again, we have decided to not implement this for now. Keeping the issue open for now to track the engagement from the community, but do not expect any updates from us Mullvad devs on this feature request in a while. Thanks for your understanding 💛
